pkgs/tools/system/netdata/default.nix

```diff
@@ -57,18 +60,20 @@ in stdenv.mkDerivation rec {
     ''}
   '';
 
-  preConfigure = optionalString (!stdenv.isDarwin) ''
+  preConfigure = optionalString withCloud ''
+    mkdir -p externaldeps
+    cp -r "${mosquitto}/lib" externaldeps/mosquitto;
+    cp -r "${libwebsockets_3_2}/lib" externaldeps/libwebsockets;
+  '' + optionalString (!stdenv.isDarwin) ''
     substituteInPlace collectors/python.d.plugin/python_modules/third_party/lm_sensors.py \
       --replace 'ctypes.util.find_library("sensors")' '"${lm_sensors.out}/lib/libsensors${stdenv.hostPlatform.extensions.sharedLibrary}"'
   '';
 
-  configureFlags = [
-    "--localstatedir=/var"
-    "--sysconfdir=/etc"
-  ];
+  configureFlags = [ "--localstatedir=/var" "--sysconfdir=/etc" ]
+    ++ optional withCloud "--enable-cloud";
 
   postFixup = ''
-    rm -r $out/sbin
```

Comment:
Why this unrelated removal?

Reply:
> Other than that, we unconditionally wrap the netdata-claim.sh script with a path to openssl and no longer remove the sbin output as it is required by the script.

Maybe it was unclear from the original message, but the included `netdata-claim.sh` script (which is used to register netdata agents to cloud) looks for netdata binaries under `sbin`. I personally thought that just keeping it would be preferable to rewriting the script.

Reply:
Right, sorry, misread that.